<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guiClass</class>
 <widget class="QWidget" name="guiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gui</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QLineEdit" name="descriptionInput">
     <property name="placeholderText">
      <string>input description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="actInput">
     <property name="placeholderText">
      <string>input act</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>add</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="acceptButton">
     <property name="text">
      <string>accept</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="savePlotButton">
     <property name="text">
      <string>save plot</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections/>
</ui>
